papers/Vvmax/RLFs.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from scipy.integrate import quad
+
+global E0
+# normalisation energy - no physical meaning, just makes units more sensible
+E0=1e22
+
+
+def make_first_guess(E,L, params,method):
+    """
+    Makes a first guess by setting the amplitude
+    to that of the first bin
+    
+    Params are always gamma,Emax
```

Comment:
define `E, L, method`
  include data type if you can

Reply:
Done

Reply:
I have defined many of these common values at the top level, since they are used throughout the file